docs/python_docs/python/tutorials/packages/gluon/transforms.md

```diff
@@ -0,0 +1,173 @@
+<!--- Licensed to the Apache Software Foundation (ASF) under one -->
+<!--- or more contributor license agreements.  See the NOTICE file -->
+<!--- distributed with this work for additional information -->
```

Comment:
All the links in this tutorial need to be updated, I'll take care of it in #16368